config/settings.yml

```diff
@@ -26,3 +26,6 @@ geohydra:
 
 purl:
   url: 'http://localhost/purl'
+
+stacks:
+  url: 'https://stacks.stanford.edu'
```

Comment:
just so no one merges yet -- I think we may have to do this somehow in shared_configs so we get stage stacks for stage ... will look into it tomorrow.

Reply:
belay that - using this PR and will tweak shared_configs after.  Better to get it working in prod now.